client/views/changelog.tpl

```diff
@@ -0,0 +1,27 @@
+{{#if latest}}
+	<h3>New version available!</h3>
+	<p>
+		There is a new {{#if latest.prerelease}}pre-{{/if}}release <b>{{latest.version}}</b> of The Lounge available.
```

Comment:
"You is a newer version of The Lounge available. You can <a>view the details on GitHub<a/>"